openLRSng/RX.h

```diff
@@ -371,26 +877,28 @@ uint8_t bindReceive(uint32_t timeout)
   init_rfm(1);
   to_rx_mode();
   Serial.println("Waiting bind\n");
-
   while ((!timeout) || ((millis() - start) < timeout)) {
     if (RF_Mode == RECEIVED) {
-      Serial.println("Got pkt\n");
+      Serial.println("Got pkt");
       len=rfmGetPacketLength();
       rfmGetPacket(rxc_buf, len);
-
-      if (rxc_buf[0] == 'b') {
-        for (uint8_t i = 0; i < sizeof(bind_data) && i < sizeof(rxc_buf); i++) {
-          *(((uint8_t*) &bind_data) + i) = rxc_buf[i + 1];
-        }
-
+      switch((char) rxc_buf[0]) {
+      case 'b': { // GET bind_data
+        // for (uint8_t i = 0; i < sizeof(bind_data); i++) {
```

Comment:
Why is this commented out? It should either be removed, or have a comment as to why it is commented out.

Reply:
oops, forgot to remove, fixed...